### DISCUSSION TITLE
Installing 10 fans in my system but not enough pwm fan headers on motherboard

### DISCUSSION THREAD

**Discussion by judelahage** [Rem0o/FanControl.Releases#815] [Q&A]
So as the title suggests I don’t have enough motherboard headers for all my fans, would I be able to use a third party pwm fan hub, like a razer, nzxt, etc or will I have to do something else. I just want to be able to individually control all the fans in my system with this software.

**Comment by snodawg** [comment]
You could probably run 2 way spitters on a couple of them.

**Reply by judelahage** [reply]
Thats not gonna cause issues with motherboard voltage and amperage right. It looks like my motherboard pwm fan headers each support a max amperage of 1 amp and each of my fans has a max current of 0.36 amps. That means if I daisy chain I can only put up to 2 fans per header without frying my motherboard.

**Reply by Rem0o** [reply]
It's a YMMV situation, depends on the mobo, and fans you are running. Seen a lot of folks use y-splitters without issues, but again, YMMV.

**Comment by Eeshen** [comment]
If you haven't seen my post (I wasn't original author, but kinda took over the post), I found that having 3 fans per motherboard header, Fan Control will not control the fans, they start running in a cycling fashion (fast-slow-fast-slow).  However, with only 2 fans per header, Fan Control appears to work ok.  
The BIOS can control 3 fans per header without issue.

https://github.com/Rem0o/FanControl.Releases/discussions/687

I hope the dev finds the issue, as Fan Control is such a GREAT program !
